randomColourModule.js

```diff
@@ -0,0 +1,73 @@
+const body = document.getElementsByTagName('body')[0]
```

Comment:
Is there a good practice to keep global scope clean?

Reply:
By global scope do you mean like window scope or just the 'global' scope of this particular module?

ES6 module variables are private by default so there is no need for patterns like this anymore:

```
(function() {
   let myPrivateVar = ....

})()
```

or 

```
return function () {
     let myPrivateVar = ....
}
```

Otherwise, these variables could be inside of a class or other object.